pkg/elle/rw_register/op.go

```diff
@@ -0,0 +1,118 @@
+package rwregister
+
+import (
+	"regexp"
+	"strconv"
+
+	"github.com/pingcap/tipocket/pkg/elle/core"
+)
+
+var (
+	opPattern = regexp.MustCompile(`([rw])([a-zA-Z])([0-9_]+)(.*)`)
+)
+
+type Int struct {
+	IsNil bool
+	Val   int
+}
```

Comment:
I prefer this struct being used by other cases, maybe it can be moved to `core` module.

Reply:
What's the difference between IntOptional and Int? Can we use IntOptional instead of creating another similar type?